expat/CMakeLists.txt

```diff
@@ -395,6 +395,12 @@ if(EXPAT_SHARED_LIBS)
     if(MSVC)
         set(expat_SRCS ${expat_SRCS} lib/libexpat.def)
     endif()
+    if(WIN32)
+        # Add DLL version
+        string(REPLACE "." "," _EXPAT_DLL_VERSION ${PROJECT_VERSION}.0)
+        add_definitions(-DVER_FILEVERSION=${_EXPAT_DLL_VERSION})
+        set(expat_SRCS ${expat_SRCS} win32/version.rc)
```

Comment:
@petitlapin I notice this file also needs listing in…

- `win32/expat.iss`
- and `_EXTRA_DIST_WINDOWS` in file `Makefile.am`

…so it gets included in release archives and people do not get compile errors.